server/src/main/java/io/crate/types/TypeSignature.java

```diff
@@ -284,4 +210,14 @@ public int hashCode() {
     public TypeSignatureType type() {
         return TypeSignatureType.TYPE_SIGNATURE;
     }
+
+    public static String escapeTypeSignatureValues(String input) {
```

Comment:
I have a some more cleanups in the pipeline which will also streamline serialization and string handling.